### ISSUE TITLE
[BUG] connection.sendDigits no longer working on 1.14.0

### ISSUE THREAD

**Issue by cmcaboy** [twilio/twilio-client.js#296]
This had been working for over 8 months for us and all of a sudden it stopped working this week. Several of the digits get repeated when inputting them. See the expected vs actual situation below. We are still on 1.14.0. The issue is not present on 2.x.x. The main thing I am looking for is if something downstream broke support for my version.

- [ x ] I have verified that the issue occurs with the latest twilio.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-client.js/blob/master/CHANGELOG.md).
 - [ x] I reviewed the [Common Issues](https://github.com/twilio/twilio-client.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ x] I verified that the [Quickstart](https://github.com/TwilioDevEd/client-quickstart-js) application works in my environment.
 - [x ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```js
device.activeConnection()?.sendDigits("1")
device.activeConnection()?.sendDigits("2")
device.activeConnection()?.sendDigits("3")
device.activeConnection()?.sendDigits("4")
```

**Expected behavior:**
The receiving end reads 1234

TODO

**Actual behavior:**

The receiving end reads 11223344

TODO

**Software versions:**

 - [ ] Browser(s): Chrome
 - [ ] Operating System: Max
 - [ ] twilio.js: 1.14.0
 - [ ] Third-party libraries (e.g., Angular, React, etc.): React

**Comment by dikirill** [comment]
Yeah, we used to have the same issue, in one of the updates it stopped, I hope it's not gonna come back again. Try to add `fakeLocalDTMF: true` into setup options.

**Comment by cmcaboy** [comment]
@dikirill I tried that, but it didn't work in my case. We ended up upgrading to `2.x`, which fixed the issue. It was just annoying as we had to deal with the breaking changes in a short period of time.
